Source/JavaScriptCore/Scripts/copy-profiling-data.sh

```diff
@@ -4,12 +4,46 @@ if [ -z "${PROFILE_DATA_FLAGS}" ]; then
     exit 0;
 fi
 
-input_profdata="${SCRIPT_INPUT_FILE_0}"
-input_profdata_type="$(file -b "${input_profdata}")"
-
+profiles="${SCRIPT_INPUT_FILE_0}"
 fallback_decompressed_profdata="${SCRIPT_INPUT_FILE_1}"
 derived_decompressed_profdata="${SCRIPT_OUTPUT_FILE_0}"
 
+# Search for a PGO profile corresponding to the current target triple or a
+# previous OS version triple. In lieu of an exact match, fall back to the most
+# recent macOS profile.
```

Comment:
I'm not quite sure how this works, given that we need to support very many old versions, which cannot fall back to latest. I guess I'll know once I see the content in WebKitAdditions.

Reply:
Old macOS versions will be handled by the second clause, "a previous OS version triple". Embedded platforms will use the latest Mac profile, like they do now.